client-app/core/utilities/date/index.ts

```diff
@@ -0,0 +1,22 @@
+export function addTimeToStartDate(dateOnly?: string): string | undefined {
```

Comment:
add comment //only accept date in format 'yyyy-MM-dd'

Reply:
Done